include/boost/format/alt_sstream_impl.hpp

```diff
@@ -255,10 +255,20 @@ namespace boost {
                 if(0 < add_size) {
                     new_size += add_size;
 #ifdef _RWSTD_NO_CLASS_PARTIAL_SPEC
+  #ifdef BOOST_NO_CXX11_ALLOCATOR
                     void *vdptr = alloc_.allocate(new_size, is_allocated_? oldptr : 0);
+  #else
```

Comment:
Is this combination possible? I would not expect the RW lib without class template partial specializations to support `allocator_traits` at all.